plugins/balloontoolbar/skins/moono/balloontoolbar.css

```diff
@@ -28,44 +22,14 @@ For licensing, see LICENSE.md or http://ckeditor.com/license
 .cke_balloon.cke_balloontoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top
```

Comment:
This selector is basically a duplicate of the two below (line 27 and 32). As the further ones overwrites it, it should be removed.